# request
string inString

---
# response 
string outString 
